### ISSUE TITLE
Track number of unique users

### ISSUE THREAD

**Issue by Popolechien** [offspot/metrics#13]
We should implement a metric that allows us to:
- Measure the number of unique devices that connected to a single Pi over time
- Report the size of the largest session (by number of users) for any single Pi

**Comment by rgaudin** [comment]
> * Report the size of the largest session (by number of users) for any single Pi

Is this the max number of simultaneous devices connected?

**Comment by Popolechien** [comment]
@rgaudin Yes. I suspect the lowest number would always be 1, and that also of less value.
